bokun.yaml

```diff
@@ -5150,6 +5150,13 @@ definitions:
       activityId:
         format: int64
         type: integer
+      isPriceConverted:
+        type: boolean
+      conversionRate:
```

Comment:
Minor note..maybe it's worth adding a description here that this is not present/null if the price is NOT converted